demo.html

```diff
@@ -39,7 +39,7 @@
     padding: 1em;
     border: 1px solid #666;
     height: 1em;
-    width: 10em;
+    width: 13em;
```

Comment:
You’re adding eleven characters to the field. Adding just 3em to the width doesn’t seem sufficient and indeed I’m seeing line breaks. What worse, since because of non-breaking spaces the line brake is with the `rgb(…)` which makes it extra awkward:
```
rgb(191, 129,
45)    #BF812D
```
Let’s just make it `20em`.

Reply:
Very sensible, doesn't look any worse either